prep-html-links/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home page</h1>
+  <a href="file:///C:/Users/Topper/Downloads/prep-html-links/contact-us/contact-us.html"> Click here to go to the contact page </a>
```

Comment:
Here you can use a relative path to the `contact-us.html` file inside of the `contact-us` directory.

```html
<a href="contact-us/contact-us.html"> ... </a>
```

Reply:
Updated. Thanks for the feedback Tim. I think I grasp it better. The examples using img src kind of threw me off.

![Prep-Html_links2](https://user-images.githubusercontent.com/90743100/134100423-1062095d-a35d-4a48-a49e-74cb0639d97a.gif)